flink-connector-oracle-cdc/src/main/java/com/ververica/cdc/connectors/oracle/util/JdbcConfigurationUtil.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.oracle.util;
+
+import java.util.Properties;
+
+/** Get the JDBC configuration utility class according to debezium properties */
+public class JdbcConfigurationUtil {
+
+    /**
+     * Gets the URL in SID format
+     *
+     * @param properties
+     * @return jdbcUrl in SID format
+     */
+    public static String getConnectionUrlWithSid(Properties properties) {
+        String url;
+        if (properties.containsKey("database.url")) {
+            url = properties.getProperty("database.url");
+        } else {
+            String hostname = properties.getProperty("database.hostname");
```

Comment:
Please check the `hostname`、`port` and `dbname` are required if user did not pass the `url` parameter.

Reply:
![image](https://user-images.githubusercontent.com/38883656/174820833-529064af-fa67-4bca-8001-9d628c23af31.png)
![image](https://user-images.githubusercontent.com/38883656/174821965-9e2be2ae-2757-484f-a1be-bf55e6391eba.png)


In fact, the hostname and port are verified at OracleSource or DebeziumSourceFunction. The URL is optional. If the URL is not configured, the hostname port database will be used to concatenate the url

Reply:
Thus the `url` config is optional and `hostname` and `port` are always required? I don't think so.

I mean we could let  `hostname` and `port` to be optional two once `url` has been configured. Please see the debezium configuration example.

https://debezium.io/documentation/reference/1.6/connectors/oracle.html#oracle-example-configuration

Reply:
Hostname and port are optional when url has been configured, otherwise hostname and port must be specified ?

Reply:
![image](https://user-images.githubusercontent.com/38883656/174938501-1c962a12-e522-43c4-9bc2-69efda00e1f3.png)

Reply:
Yes, you can add test to cover this.  A url is enough and more simple in most cases from user's perspective,

Reply:
ok

Reply:
Some time was delayed due to work, please look at it now